doc-content/enterprise-only/installation/run-dc-standalone-proc.adoc

```diff
@@ -16,58 +16,37 @@ This JAR file is supported only when it is run on Red Hat Enterprise Linux.
 . Extract the downloaded `{PRODUCT_FILE}-addons.zip` to a temporary directory. This archive includes the `{PRODUCT_FILE}-standalone-sample-configuration.zip` file.
 . Extract the `{PRODUCT_FILE}-standalone-sample-configuration.zip` file to the directory that contains the `{PRODUCT_FILE}-{URL_COMPONENT_CENTRAL}-standalone.jar` file. The `{PRODUCT_FILE}-standalone-sample-configuration.zip` file contains the following sample configuration files:
 +
-* `application-roles.properties`
-* `application-users.properties`
-* `sample-standalone-config.yml`
+* `application-script.cli` - Sample script for adding a user and kie server system properties
```

Comment:
I would use a colon instead of a dash:
`application-script.cli`: Sample script for adding a user and kie server system properties